### PR TITLE
Update Blogtrottr.com.xml

### PR DESCRIPTION
**Pull request by terrorist96**
It works for me right now.

### CODE REVIEW

File: src/chrome/content/rules/Blogtrottr.com.xml [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 	- li.blogtrottr.com, equivalent to p.liadm.com
 -->
 
 <ruleset name="Blogtrottr.com">
 	<target host="blogtrottr.com" />
 	<target host="www.blogtrottr.com" />
+	<target host="blog.blogtrottr.com" />
 	<target host="li.blogtrottr.com" />
 	<target host="static.blogtrottr.com" />
 	<target host="support.blogtrottr.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Blogtrottr.com.xml:12]:
```suggestion
	<target host="blogtrottr.com" />
	<target host="www.blogtrottr.com" />
	<target host="api.blogtrottr.com" />
	<target host="api-test.blogtrottr.com" />
	<target host="beta.blogtrottr.com" />
	<target host="blog.blogtrottr.com" />
	<target host="lb.blogtrottr.com" />
	<target host="li.blogtrottr.com" />
	<target host="preview.blogtrottr.com" />
	<target host="prtg.blogtrottr.com" />
	<target host="staging.blogtrottr.com" />
	<target host="static.blogtrottr.com" />
	<target host="static-test.blogtrottr.com" />
	<target host="support.blogtrottr.com" />
	<target host="web-test.blogtrottr.com" />
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

**Comment by bfosterscripps** [comment]
[Job failed](https://travis-ci.org/github/EFForg/https-everywhere/jobs/731598146) because there was an issue with `prtg.blogtrottr.com`:

> ERROR src/chrome/content/rules/Blogtrottr.com.xml: Fetch error: http://prtg.blogtrottr.com/ => https://prtg.blogtrottr.com/: (60, 'SSL certificate problem: unable to get local issuer certificate')

File: src/chrome/content/rules/Blogtrottr.com.xml [hunk 2]
```diff
@@ -1,17 +1,24 @@
 <!--
-	Connection closed:
-	- blog.blogtrottr.com
-
 	Invalid certificate:
 	- li.blogtrottr.com, equivalent to p.liadm.com
 -->
```
Comment (pipboy96) [inline src/chrome/content/rules/Blogtrottr.com.xml:4]:
```suggestion
	Invalid certificate:
	- li.blogtrottr.com, equivalent to p.liadm.com

	Incomplete certificate chain:
	- prtg.blogtrottr.com
-->
```

File: src/chrome/content/rules/Blogtrottr.com.xml [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
 	<target host="blogtrottr.com" />
 	<target host="www.blogtrottr.com" />
+	<target host="api.blogtrottr.com" />
+	<target host="api-test.blogtrottr.com" />
+	<target host="beta.blogtrottr.com" />
+	<target host="blog.blogtrottr.com" />
+	<target host="lb.blogtrottr.com" />
 	<target host="li.blogtrottr.com" />
+	<target host="preview.blogtrottr.com" />
+	<target host="prtg.blogtrottr.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Blogtrottr.com.xml:16]:
```suggestion
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

**Review by pipboy96 — APPROVED** [review]

**Pull request merged**